### ISSUE TITLE
Add a license 

### ISSUE THREAD

**Issue by mattkanwisher** [weaver/moniker#3]

**Comment by carles** [comment]
Indeed, it would be nice if you could provide a license for this package. As I understand it (and I'm not a lawyer) without a license it defaults to being copyrighted and technically cannot be used by other people.

I suspect this is not what you intended since the project has been publicly available on npm for the last seven years and it's also a dependency of many other projects which use an open-source license as well (which technically puts them in the wrong  :P )

I would suggest ISC (https://opensource.org/licenses/ISC) which is the default one that npm uses.
